readthedocs.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+build:
+  image: latest
+
+formats: []
+
+python:
+  version: 3.7
+  install:
+    - requirements: docs.txt
```

Comment:
Could we just list the requriements here in this file rather than in a separate docs.txt? That at least would be an improvement since then depedencies are always in yaml files in all cases, not plain text files.  Also, can this be `.readthedocs.yml`? Trying to reduce clutter a the top level for new devs

Reply:
> Could we just list the actual requirements here in this file rather than in a separate docs.txt?

No, yet. We do have an issue open to support specifying dependencies in this file: https://github.com/readthedocs/readthedocs.org/issues/5572

> Also, can this be `.readthedocs.yml`?

Yes.